lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -254,13 +280,7 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
 
         this.startProcessCloudService
```

Comment:
When the API call for definitions fails, will we display the running spinner infinitely?

Reply:
Probably yes 🤷‍♀️. As already talked with Maurizio, this component really needs to be rewritten. I only focused on a bug fix itself and some minor non-breaking improvements

Reply:
thx, sure, resolved